lib/src/text_controller.dart

```diff
@@ -51,18 +51,20 @@ class _TextEditingControllerHookCreator {
 ///
 /// See also:
 /// - [TextEditingController], which this hook creates.
-const useTextEditingController = _TextEditingControllerHookCreator();
```

Comment:
Const is important. It is a tiny bit faster than using final. It's negligible, but code-coverage for this is negligible too.